docs/adrs/003-Retry_endpoint_design.md

```diff
@@ -35,26 +35,26 @@ The endpoint takes a job or list of job ids and then attempts to resubmit them t
 * EE2 looks up the job versions and parameters, and then submits the job to be retried, incrementing the `retry_count`
   of the job being retried, and the newly launched job gains a pointer to the `_PARENT_RETRY_JOB_ID`
 * The job is submitted and upon successful submission, notifies the `retry_parent` and notifies the `parent_job_id` that a new `child_job` has been added
```

Comment:
the job doesn't notify anything itself -- ee2 updates the parent job with the new child job ID and updates the retry count of the job that has been retried

Reply:
Updated The job is submitted and upon successful submission, the child job adds the field retry_parent and notifies the parent_job_id that a new child_job has been added by appending itself to the parent_job.child_jobs[] field